Tools/TestWebKitAPI/Tests/WebKitCocoa/WKWebExtensionAPITabs.mm

```diff
@@ -1422,6 +1487,187 @@
     [manager loadAndRun];
 }
 
+TEST(WKWebExtensionAPITabs, ExecuteScript)
+{
+    TestWebKitAPI::HTTPServer server({
+        { "/"_s, { { { "Content-Type"_s, "text/html"_s } }, "<iframe src='/frame.html'></iframe>"_s } },
+        { "/frame.html"_s, { { { "Content-Type"_s, "text/html"_s } }, "<html><body onload=\"document.body.style.background ='blue'\"/></html>"_s } },
```

Comment:
can change this to `"<body style='background-color: blue'></body>"` to match the other tests